Abc.MoqComplete/Abc.MoqComplete/Services/AutoMockerMethodIdentifier.cs

```diff
@@ -0,0 +1,44 @@
+﻿using JetBrains.ProjectModel;
+using JetBrains.ReSharper.Psi.CSharp.Tree;
+
+namespace Abc.MoqComplete.Services
+{
+	[SolutionComponent]
+	public class AutoMockerMethodIdentifier : BaseMethodIdentifier, IAutoMockerMethodIdentifier
```

Comment:
minor: I would have added all the below methods directly to MoqMethodIdentifier; but it ok to keep it like this

Reply:
I merge  from `AutoMockerMethodIdentifier` method 'IsAutoMockerSetupMethod' into `MoqMethodIdentifier` and remove `AutoMockerMethodIdentifier` and `BaseMethodIdentifier`.
PR will be updated later